Base/QTGUI/qSlicerModulesMenu.cxx

```diff
@@ -39,8 +39,8 @@ class qSlicerModulesMenuPrivate
   void init();
   void addDefaultCategories();
 
-  void addModuleAction(QMenu* menu, QAction* moduleAction, bool useIndex = true);
-  QMenu* menu(QMenu* parentMenu, QStringList subCategories);
+  void addModuleAction(QMenu* menu, QAction* moduleAction, bool builtIn = true, bool useIndex = true);
```

Comment:
The behavior is different for the same inputs with the old and new implementation (if only one optional parameter is defined then it's the useIndex in earlier versions but builtIn in the new one). If we keep the useIndex argument as the third argument then there is no backward compatibility issue.